.github/workflows/test-golang.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
         # action @v1.3.0 can install protoc 3.19, @v2 installs protoc v21.0+
         uses: arduino/setup-protoc@v1.3.0
         with:
-          version: "3.19.6"
+          version: "3.20.2"
```

Comment:
it would be better to move this, and the changes that it causes, to it's own commit

Reply:
Done.